packages/rocketchat-theme/client/imports/general/base_old.css

```diff
@@ -4432,6 +4432,45 @@ body:not(.is-cordova) {
 	color: #a9a9a9;
 }
 
+.global-announcement {
+	padding: 20px;
+	width: 100%;
+	margin: auto;
+	position: relative;
+	line-height: 140%;
+	z-index: 1;
+
+	& .button {
+	  float: right;
+	  padding: 6px 6px;
+	  top: -6px;
+	  right: -16px;
+  	}
+	box-shadow:
+		0 1px 1px 0 rgba(0, 0, 0, 0.2),
+		0 2px 10px 0 rgba(0, 0, 0, 0.16);
+}
+
+/* on Login-screen */
+.wrapper .global-announcement {
```

Comment:
maybe, dedicated classes should be used. However, I wanted to modify as least as possible